exist-parent/pom.xml

```diff
@@ -1177,17 +1177,28 @@
     </reporting>
 
     <repositories>
-        <repository>
```

Comment:
Please leave the existing repository in place as it is needed for a number of purposes including Nightly Builds and Releases.

Reply:
```shell
The project org.exist-db:exist:7.0.0-SNAPSHOT (/Users/hal1000/Documents/GitHub/exist/pom.xml) has 1 error
[ERROR]     Non-parseable POM /Users/hal1000/Documents/GitHub/exist/exist-parent/pom.xml: Duplicated tag: 'snapshotRepository' 
```

can your automation not pull the snapshots from the GitHub package registry, these builds will be automated?